packages/containers/src/SidePanel/SidePanel.test.js

```diff
@@ -19,42 +22,226 @@ describe('SidePanel', () => {
+				href: 'whatever/path/to/route',
+				onClick: expect.anything(),
+				onClickDispatch: {
+					type: ACTION_TYPE_LINK,
+					cmf: {
+						routerPush: 'whatever/path/to/route',
+					},
+				},
+			});
+			expect(props.actions[1].onClick).toBeUndefined();
+		});
+
+		cases(
+			'should define "selected" action based on the same "menuActions" item "path"',
+			({ currentRoute, itemRoute, isMatching }) => {
+				state.routing.locationBeforeTransitions.pathname = currentRoute;
+				const actionSelectable = {
+					identity: 'The one',
+					href: itemRoute,
+				};
+
+				const props = mapStateToProps(state, {
+					componentId: uuid.v4(),
+					actions: [
+						{
+							anotherRandomSpecificActionProp: 'other prop',
+							href: 'whatever/path/to/route',
+						},
+						actionSelectable,
+					],
+				});
+
+				if (isMatching) {
+					expect(props.selected).toMatchObject(actionSelectable);
+				} else {
+					expect(props.selected).toBeUndefined();
+				}
+			},
+			[
+				{
+					name: 'Usual simple path',
+					currentRoute: '/a/usual/path',
+					itemRoute: '/a/usual/path',
+					isMatching: true,
+				},
+				{
+					name: 'Usual simple path with trainling /',
+					currentRoute: '/a/usual/path/',
+					itemRoute: '/a/usual/path',
+					isMatching: true,
+				},
+				{
+					name: 'Usual simple path with trainling / on match',
+					currentRoute: '/a/usual/path',
+					itemRoute: '/a/usual/path/',
+					isMatching: true,
+				},
+				{
+					name: 'Subset path starting at the beginning',
+					currentRoute: '/a/usual/path',
+					itemRoute: '/a/usual',
+					isMatching: true,
+				},
+				{
+					name: 'Complex subset path starting at the beginning',
+					currentRoute: '/some/path/not/super/really/usual/but/which/can/occur',
+					itemRoute: '/some/path/not/super/really/usual',
+					isMatching: true,
+				},
+				{
+					name:
+						'Subset path starting at the beginning but without the exact last segment (currentRoute longer)',
+					currentRoute: '/a/usual/path',
+					itemRoute: '/a/usual/pat',
+					isMatching: false,
+				},
+				{
+					name:
+						'Subset path starting at the beginning but without the exact last segment (currentRoute shorter)',
+					currentRoute: '/a/usual/pat',
+					itemRoute: '/a/usual/path',
+					isMatching: false,
+				},
+				{
+					name:
+						'Subset path starting at the beginning but without the last item path segment matching (currentRoute with more segments)',
+					currentRoute: '/a/usual/path/with/some/others/paths',
+					itemRoute: '/a/usual/pat',
+					isMatching: false,
+				},
+				{
+					name: 'Subset path but not starting at the beginning',
+					currentRoute: '/a/usual/path',
+					itemRoute: '/usual/path',
+					isMatching: false,
+				},
+				{
+					name: 'Just a total other path',
+					currentRoute: '/a/usual/path',
+					itemRoute: '/this/path/has/nothing/in/common',
+					isMatching: false,
+				},
+				{
+					name: 'More specific path than the current route',
```

Comment:
Same here. In a generic way, the feature is that it should match url that starts with the one we set. we should test 3 cases : 
* it starts with
* it doesn't starts with
* edge case : it starts with but the last word doesn't match